test/e2e/scenarios/dns/scenarios.go

```diff
@@ -201,7 +203,7 @@ func ValidateAdvancedDNSMetrics(scenarioName string, req *RequestValidationParam
 		},
 		{
 			Step: &ValidateAdvancedDNSRequestMetrics{
-				Namespace:          "kube-system",
+				Namespace:          common.Namespace,
```

Comment:
For the advanced metrics check, we should be checking in the retina pod namespace, right? 🤔 @matmerr